### PR TITLE
Represent 0 using log(1.0)

### PR DESCRIPTION
**Pull request by jiradeto**
Related issue: #426 

Add a new case to represent 0 as the log() function of the opaque one expression.

### CODE REVIEW

File: generator/src/main/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGenerator.java [hunk 1]
```diff
@@ -199,6 +199,16 @@ private Expr makeOpaqueZeroOrOne(boolean isZero, BasicType type, boolean constCo
           }
           return new FunctionCallExpr("exp", makeOpaqueZero(type, constContext, newDepth,
               fuzzer));
+        case 6:
+          // represent 0 as the natural logarithm of opaqueOne, e.g. log(1.0)
+          if (!isZero) {
+            continue; // log(opaqueZero) only provides a means of representing 0, not 1
```
Comment (afd) [inline generator/src/main/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGenerator.java]: Should this say "log(opaqueOne) only provides ..."?

**Comment by afd** [comment]
The CI fails, but it is due to this existing issue that (due to the wonders of randomization) your PR happens to un-mask:

https://github.com/google/graphicsfuzz/issues/435

I have a fix for that issue, so let's get that in first.

**Comment by jiradeto** [comment]
Oh, I was inspecting to the failures. Thanks for your informed!

**Comment by afd** [comment]
I have fixed #435, so can you please rebase your changes?  I'm hopefully that the CI will then go through.

File: generator/src/main/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGenerator.java (hunk 1 above)
Comment (jiradeto) [reply generator/src/main/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGenerator.java]: Fixed

**Comment by jiradeto** [comment]
Done!

**Pull request merged**